### ISSUE TITLE
Suggestion: Village Boarder markers

### ISSUE THREAD

**Issue by Magleose** [TangoTek/TekTopia-Community#232]
Today I spent a bit of time trying to establish village borders. It would be cool if we had an indication or village border marker, maybe even something we can drop on the ground to lay out the village structures. I have been using and moving a guard post to mark the edges of valid village.  I know it can be counted out, but something physical could be neat to have in the toolbox

**Comment by JohnyDL** [comment]
Yeah, it's 161 block square unless you're chunk aligned with the village hall it's not easy to quickly count 5 chunks in each direction. (might be a job for the architect give him banners in the storage room and he goes to mark out the village?)

**Comment by Magleose** [comment]
that would be amazing

**Comment by forthwist** [comment]
Is this what that forcefield is?   I moved into a castle near town and discovered it cutting through my base.  D:    Is there a way to toggle it off?
